data/contributor_data.json

```diff
@@ -0,0 +1,3 @@
+{
+  
+}
```

Comment:
This file should be deleted. It might cause a merge conflict with the master branch but it's in there by mistake